spec.html

```diff
@@ -9102,11 +9103,11 @@ <h1>StringGetOwnProperty ( _S_, _P_ )</h1>
           1. Let _index_ be ! CanonicalNumericIndexString(_P_).
           1. If _index_ is *undefined*, return *undefined*.
           1. If IsInteger(_index_) is *false*, return *undefined*.
-          1. If _index_ is *-0*, return *undefined*.
+          1. If _index_ is *-0*<sub>𝔽</sub>, return *undefined*.
           1. Let _str_ be _S_.[[StringData]].
           1. Assert: Type(_str_) is String.
           1. Let _len_ be the length of _str_.
-          1. If _index_ &lt; 0 or _len_ &le; _index_, return *undefined*.
+          1. If _index_ &lt; *0*<sub>𝔽</sub> or _len_ &le;<sub>𝔽</sub> _index_, return *undefined*.
```

Comment:
There is no such thing as *0*<sub>𝔽</sub>. There should be no occurrences of this anywhere. @bakkot linting rule?

Reply:
Added a desiderata.